frontend/src/bundles/studio/types/voice.type.ts

```diff
@@ -0,0 +1,8 @@
+type Voice = {
+    id: string;
+    name: string;
+    previewUrl?: string;
+    code: string;
+};
```

Comment:
We already have voice type and it is not the same as you created. Make sure to use only those properties that exist in endpoint response.
Please review this pr https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/185 to reuse Voice type and review what properties are returned.